x-pack/platform/plugins/shared/security/public/authentication/login/components/login_form/login_form.tsx

```diff
@@ -130,9 +130,18 @@ const assistanceCss = (theme: UseEuiTheme) => css`
   }
 `;
 
+const noProvidersMessage = i18n.translate('xpack.security.noAuthProvidersForDomain', {
+  defaultMessage: 'No authentication providers have been configured for this domain.',
```

Comment:
__question:__ It feels like `domain` is a bit confusing in this context since it's not exactly the same thing as `origin`, and also there might be IP-based origins. What about changing this to something like this?
```suggestion
  defaultMessage: 'No authentication providers have been configured for this origin ({origin}).',
```

where the `origin` parameter is the current origin that the user can then copy and paste into the configuration if needed?